src/tests/eth_connector.rs

```diff
@@ -173,10 +172,9 @@ fn get_eth_on_near_balance(master_account: &UserAccount, acc: &str, contract: &s
         "ft_balance_of",
         json!({ "account_id": acc }).to_string().as_bytes(),
     );
-    String::from_utf8(balance.unwrap())
-        .unwrap()
-        .parse()
-        .unwrap()
+    let val_str = String::from_utf8(balance.unwrap()).unwrap();
+    let val = &val_str[1..val_str.len() - 1];
+    val.parse().unwrap()
```

Comment:
Btw, this is, of course, the scope of the #241 PR, but I think that things like that need to be wrapped up into another function. Otherwise, it'll be hard both for us and external contributors to get what's the reason why we're doing such weird slices

Reply:
Thats because of a weird quirk of Rust. It wouldn't let me actually remove the quotations through usual methods. I have no answer for why, but that was the only method through 5 others that I tried that actually worked in the end.